versions.tf

```diff
@@ -7,6 +7,11 @@ terraform {
       version = ">= 4.0.0"
     }
 
+    null = {
+      source  = "hashicorp/null"
+      version = ">= 3.0.0"
```

Comment:
Should this be here? Where are you using the null resource on the module?

If you are using it on the examples keep it there but not here.

Reply:
@hugoalmeida264  There is an option to download created aws key pair to local using null resource and local-exec provisioner